daemon/c_service.c

```diff
@@ -555,7 +603,7 @@ c_service_send_message(c_service_t *service, c_service_message_t message)
 
 	case C_SERVICE_MESSAGE_RESUME:
 		if ((ret = c_service_send_message_proto(service,
-							CMLD_TO_SERVICE_MESSAGE__CODE__RESUME)) < 0)
+							CMLD_TO_SERVICE_MESSAGE__CODE__RESUME) < 0))
```

Comment:
accidentally changed? we want to check ret < 0 and not (ret  = ... < 0)  which is always true.

Reply:
yes, changed it back
thx!